xcell/cls/cl.py

```diff
@@ -226,12 +226,31 @@ def get_cl_file(self):
                 cl -= nl
             # Note that while we have subtracted the noise
             # bias from `cl_cp`, `cl_cov_cp` still includes it.
+            correction = None
+            if (mapper1.mask_power > 1) or (mapper2.mask_power > 1):
+                # Applies correction factor if masks have been
+                # implicitly applied to the maps
+                # See ACTk for reference
+                n_a = mapper1.mask_power
+                n_b = mapper2.mask_power
+                w_a = mapper1.get_mask()
+                w_b = mapper2.get_mask()
+                # Currently this first step is done with the pixell masks
```

Comment:
Can you explain what you mean by this comment?

Reply:
it referred to the fact that we need use the pixell mask to undo the mask factor. 
I am going to remove it becuase it no longer applies as this step is done at the mapper level